### PR TITLE
Init meeting notes

### PR DESCRIPTION
**Pull request by ElijahCaine**
Added the initial meeting notes and some tweaks to the docs so they work properly.

*It should be noted that I didn't review the notes from our meeting thoroughly and I may have missed something or written in the wrong tone. Feedback is welcome.*

### CODE REVIEW

**Comment by Kennric** [comment]
Looks good to me. +1

File: docs/source/meetings/dev_meeting_2015-04-09.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+* Submit Time
+    * project name; slug/fuzzy (str)
+    * duration: minutes (int)
+    * user: ~some auth backend~ username (str) perhaps SSH keys, oauth
+    * activity: slig (str/null)
+    * notes: notes (str)
+    * issue_ur: uri (str)
+    * date_worked: iso standard date (str)
+* project list
+* activity lst
```
Comment (tschuy) [inline docs/source/meetings/dev_meeting_2015-04-09.rst]: list

File: docs/source/meetings/dev_meeting_2015-04-09.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+Evan drew some stuff on the whitebord.
+
+* Submit Time
+    * project name; slug/fuzzy (str)
+    * duration: minutes (int)
+    * user: ~some auth backend~ username (str) perhaps SSH keys, oauth
+    * activity: slig (str/null)
+    * notes: notes (str)
+    * issue_ur: uri (str)
```
Comment (tschuy) [inline docs/source/meetings/dev_meeting_2015-04-09.rst]: issue_uri

File: docs/source/meetings/dev_meeting_2015-04-09.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    * werkzug - very barebones
+    * flask - slightly more than werkzug, some batteries included
+    * django - all the stuff
+        * django rest-api
+    * sandman - flask + a little more (dean's proposition)
+    * go? (this caused a big debate)
+    * javascript? (nodejs + express)
+    * **Radical Idea** pair off and quickly write/demo an something. 1week till
+      check-in, decide by 3weeks. 50% time working on this.
+4. Explore JS implementation (Dean + Lucy + Eli + Ian)
```
Comment (tschuy) [inline docs/source/meetings/dev_meeting_2015-04-09.rst]: + Evan

**Comment by Kennric** [comment]
+1

**Pull request merged**